api/src/pcapi/local_providers/provider_manager.py

```diff
@@ -176,7 +187,8 @@ def collect_elligible_venues_and_activate_ems_sync() -> None:
         price_rules_to_delete: list[int] = []
 
         ems_provider_id = providers_repository.get_provider_by_local_class("EMSStocks").id
-        available_sites_by_allocine_id = {venue.allocine_id: venue for venue in connector.get_available_sites()}
+        available_sites_by_allocine_id = {site.allocine_id: site for site in connector.get_available_sites()}
+        available_sites_by_cinema_id = {site.id: site for site in connector.get_available_sites()}
```

Comment:
```suggestion
        sites = connector.get_available_sites()
        available_sites_by_allocine_id = {site.allocine_id: site for site in sites}
        available_sites_by_cinema_id = {site.id: site for site in sites}
```
Pour éviter un double appel HTTP

Reply:
/facepalm bien vu merci